README.md

```diff
@@ -0,0 +1,10 @@
+# Catalogue Charmed Operator for Kubernetes
+
+[![Charmhub Badge](https://charmhub.io/catalogue-k8s/badge.svg)](https://charmhub.io/catalogue-k8s)
+[![Release Edge](https://github.com/canonical/catalogue-k8s-operator/actions/workflows/release-edge.yaml/badge.svg)](https://github.com/canonical/catalogue-k8s-operator/actions/workflows/release-edge.yaml)
+[![Release Libraries](https://github.com/canonical/catalogue-k8s-operator/actions/workflows/release-libs.yaml/badge.svg)](https://github.com/canonical/catalogue-k8s-operator/actions/workflows/release-libs.yaml)
+[![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
```

Comment:
For consistency, should these be in the `./charm/README.md`?

Reply:
I will ALSO add them there if you want, the reason they are here is to simplify getting data for the future release page :D